package.json

```diff
@@ -61,7 +61,8 @@
     "plugin:build": "nx run-many -t build --projects='@grafana-plugins/*' --exclude \"@grafana-plugins/input-datasource\"",
     "plugin:build:commit": "nx run-many -t build:commit --projects='@grafana-plugins/*' --exclude \"@grafana-plugins/input-datasource\"",
     "plugin:build:dev": "nx run-many -t dev --projects='@grafana-plugins/*' --exclude \"@grafana-plugins/input-datasource\"",
-    "generate-icons": "yarn workspace @grafana/saga-icons generate"
+    "generate-icons": "yarn workspace @grafana/saga-icons generate",
+    "generate-apis": "rtk-query-codegen-openapi ./scripts/generate-rtk-apis.ts"
```

Comment:
how long does this take to run? can we run it as a precommit hook eventually? 🤔

Reply:
it's pretty quick. We could run it with precommit hook... or it could run as a part of the step that actually generates the openapi files in the first place.... or it could run as a dependency with nx.

i very deliberatly decided not to solve that right now 😅